src/articles/_drafts/tv-snap-mrss-requirements.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Opera TV Snap data format requirements
+authors:
+- oskar-furga
+intro: 'Specification of Opera MRSS format required to publish applications using TV Snap Template.'
+tags:
+- mrss
+- snap
+- store
+- template
+- tv
+- video
+license: cc-by-3.0
+---
+
+## Table of Contents
+
+- [Introduction](#introduction)
+    - [Basics](#basics)
+- [Opera MRSS](#mrss)
+    - [Feed example](#feed)
+    - [Item example](#item)
+    - [Visualization](#visualization)
+
+## Introduction {#introduction}
+
+Opera TV Snap is an industry-first, end-to-end solution that allows content publishers to create and submit a Smart TV app into the Opera TV Store quickly and easily, for free. See [opera.com/tvsnap][1] for more details.
+
+### Basics {#basics}
+
+Specification you should be aware before preparing video feed for TV Snap Template:
```

Comment:
- Specifications you should be aware of before preparing a video feed for the TV Snap Template: